crawler-cli/src/main/java/org/gbif/crawler/dwca/avroconverter/DwCAToAvroConverterCallBack.java

```diff
@@ -0,0 +1,21 @@
+package org.gbif.crawler.dwca.avroconverter;
+
+import org.gbif.common.messaging.AbstractMessageCallback;
+import org.gbif.common.messaging.api.messages.DwcaDownloadFinishedMessage;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Call back which is called when the {@link org.gbif.common.messaging.api.messages.DwcaDownloadFinishedMessage } is received.
+ * ToDo Implement DwCAToAvro Conversion
+ */
+public class DwCAToAvroConverterCallBack extends AbstractMessageCallback<DwcaDownloadFinishedMessage> {
+
+  private static final Logger LOG = LoggerFactory.getLogger(DwCAToAvroConverterCallBack.class);
+
+  @Override
+  public void handleMessage(DwcaDownloadFinishedMessage dwcaDownloadFinishedMessage) {
+    LOG.info("Received Download finished message, " + dwcaDownloadFinishedMessage.toString());
```

Comment:
This should listen for the validation report, not the download finished.  This is documented in the spec on https://github.com/gbif/pipelines/issues/59